### DISCUSSION TITLE
discrete and continuous clusters

### DISCUSSION THREAD

**Discussion by wangjiawen2013** [theislab/scvelo#460] [Q&A]
Hi,
Are there any prerequsites to use scvelo ?
It seems that the tutorial of scvelo uses continuous cell states,:
![image](https://user-images.githubusercontent.com/29703450/111562422-27c3f380-87d1-11eb-80bd-da57e549707c.png)

what about discrete states ? for example:
![QQ图片20210318100751](https://user-images.githubusercontent.com/29703450/111562784-bdf81980-87d1-11eb-8db5-a5e250908365.png)
In this case, the velocity stream is broken becasue of the discrete cell clusters,  then how to explain the RNA velocity ?

**Comment by VolkerBergen** [comment]
The actual velocities as well as the transition probabilities are independent of the embedding (checkout e.g. https://github.com/theislab/cellrank making use of those in high-dim space), while the projection is embedding-dependent of course. Yet, if the UMAP coord with reasonable parameter settings show such discrete clusters, you would also expect them to be rather distinct not expecting any observable transitions between them.

It could happen though, that in perturbational data some gene-specific response cause the discreteness, while you may observe some continuous transitions in particular genes. In such a scenario, there's no way around investigating individual gene dynamics.

**Reply by WeilerP** [reply]
@wangjiawen2013, does this answer your question? If so, please mark the discussion as answered.
